sdk/go/pulumi/generate.go

```diff
@@ -58,8 +58,7 @@ func (b builtin) DefineInputMethods() bool {
 
 func (b builtin) ImplementsPtrType() bool {
 	switch b.Type {
-	case "bool", "float32", "float64", "ID", "int", "int16", "int32", "int64", "int8", "string", "URN", "uint",
-		"uint16", "uint32", "uint64", "uint8":
+	case "bool", "float64", "ID", "int", "string", "URN":
```

Comment:
Should we keep `uint` around as well? Seems like we might want at least one unsigned type.

Reply:
I'd bias to not add any types unless we have a concrete use case driving it. Remember that these base types generate an explosion of map types, array types, pointer types, etc as well. Fairly easy to add back later if need be.

Reply:
Fair enough. LGTM